code-challenges.js

```diff
@@ -24,12 +49,63 @@ var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
 
+const middle = (array) => {
+ let splitted = array.split("")
+//  even number of letters
+ if (splitted.length %2==0)
+// probably string interpolation is not the best way to do it but it works :)
+ return `${splitted[Math.floor((splitted.length / 2)-1)]}${splitted[Math.floor(splitted.length / 2)]}`
+// odd numbers of letters
+else if (splitted.length %2 !==0)
+return splitted[Math.floor(splitted.length / 2)]
+}
+
+console.log("Challenge 3.1:",middle(middleLetters1))
+console.log("Challenge 3.2:",middle(middleLetters2))
+
+// bonus variable to double check
+// var middleLetters3 = "abstract"
+// // Expected output: “tr”
+// console.log("Challenge 3.3:",middle(middleLetters3))
 
```

Comment:
Logic looks solid. Be careful with your curly braces. They seem to be a little wild here.